External/FEXCore/Source/Interface/Core/JIT/Arm64/JIT.cpp

```diff
@@ -336,6 +370,40 @@ Arm64JITCore::Arm64JITCore(FEXCore::Context::Context *ctx, FEXCore::Core::Intern
   : Arm64Emitter(ctx, 0)
   , CTX {ctx}
   , ThreadState {Thread} {
+
+  {
+    // Set up pointers that the JIT needs to load
+    auto &Pointers = ThreadState->CurrentFrame->Pointers.AArch64;
+    // Process specific
+    Pointers.LUDIV = reinterpret_cast<uint64_t>(LUDIV);
+    Pointers.LDIV = reinterpret_cast<uint64_t>(LDIV);
+    Pointers.LUREM = reinterpret_cast<uint64_t>(LUREM);
+    Pointers.LREM = reinterpret_cast<uint64_t>(LREM);
+    Pointers.PrintValue = reinterpret_cast<uint64_t>(PrintValue);
+    Pointers.PrintVectorValue = reinterpret_cast<uint64_t>(PrintVectorValue);
+    Pointers.RemoveCodeEntryFromJIT = reinterpret_cast<uintptr_t>(&Context::Context::RemoveCodeEntryFromJit);
+    Pointers.CPUIDObj = reinterpret_cast<uint64_t>(&CTX->CPUID);
+
+    {
+      using ClassPtrType = FEXCore::CPUID::FunctionResults (FEXCore::CPUIDEmu::*)(uint32_t, uint32_t);
```

Comment:
One minor note:

> The code produced the expected results for you since the tested platforms put the function address in the first 8 bytes of the PMF.

If we keep the PMF type-punning, the latter 8 bytes in the PMF data should be zeroed out explicitly. Currently, we risk a crash by not initializing them at all, I think.

Reply:
Upon more thought, for this particular case I think this should be good enough:

```cpp
// Extract function address from PMF. This is specific to platform ABI, but Itanium/WhateverTheEquivalentIsOnAarch64 put the function address in the first 8 bytes.
struct PointerToMember {
    uintptr_t func;
    uintptr_t off;
} pmf;
static_assert(sizeof(pmf) == sizeof(&FEXCore::CPUIDEmu::RunFunction));
memcpy(&pmf, &FEXCore::CPUIDEmu::RunFunction, sizeof(pmf));
assert(pmf.off == 0);
Pointers.CPUIDFunction = pmf.func;
```

Reply:
Added a utility class to do conversion of these for us with validation that we don't try converting a virtual member function.
Also ensures correctness with the difference between Itanium and AArch64 C++ABI differences for these.
Resolved now.